libsolidity/ast/Types.h

```diff
@@ -1286,6 +1286,7 @@ class FunctionType: public Type
 		ABIEncodeWithSignature,
 		ABIDecode,
 		GasLeft, ///< gasleft()
+		BlobHash, ///< BLOBHASH
 		MetaType, ///< type(...)
 		/// Refers to a function declaration without calling context
```

Comment:
Yeah, I wasn't sure if this change was breaking anything, but since it didn't cause any test failures, I assumed it wasn't. However, I'm not certain if there's something that relies on the exact enum value.

Reply:
This is still not done. In the current form the IDs are changing anyway so we can just as well put it next to `blockash`.

Reply:
Done.